README.md

```diff
@@ -9,17 +9,30 @@
 ![Singularity Container available](
 https://img.shields.io/badge/singularity-available-7E4C74.svg)
 
-
-
 ### Introduction
 
 **nf-core/methylseq** is a bioinformatics best-practice analysis pipeline used for Methylation (BS-Seq) data analysis.
 
 The pipeline uses [Nextflow](https://www.nextflow.io), a bioinformatics workflow tool. It pre-processes raw data from FastQ inputs, aligns the reads and performs extensive quality-control on the results.
 
-### Choice of workflows
+### Pipeline Steps
+
+The pipeline allows you to choose between running either [Bismark](https://github.com/FelixKrueger/Bismark) or [bwa-meth](https://github.com/brentp/bwa-meth) / [MethylDackel](https://github.com/dpryan79/methyldackel).
+Choose between workflows by using `--aligner bismark` (default) or `--aligner bwameth`.
+
+| Step   | Bismark Workflow | bwa-meth Workflow |
```

Comment:
Just one last comment, table doesn't look good in plain text, but I guess it's difficult to align everything

Reply:
Yeah, I'll redo it now. I was offline when I wrote this but can redo it using an online tool, then it's easy.